crates/proto/src/op/edns.rs

```diff
@@ -56,6 +58,30 @@ impl Edns {
         Self::default()
     }
 
+    /// Creates a new extended DNS object prepared for DNSSEC messages.
```

Comment:
According to `git bisect`, this commit is causing test failures. I'm not seeing the difference...

Reply:
The issue is that requests in these tests already have an `OPT` record set earlier in the client, (setting the maximum payload length) so calling `get_or_insert_with()` on the `Option<Edns>` does nothing, and thus misses filling in the DAU/DHU options.

Reply:
Ahh, thanks!